resources/lib/findologic_client.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+require_once __DIR__ . '/ApiResponse.php';
+use FINDOLOGIC\Api\Client;
+use FINDOLOGIC\Api\Config;
+use FINDOLOGIC\Api\Requests\Request;
+use FINDOLOGIC\Api\Definitions\OutputAdapter;
+use FINDOLOGIC\Api\Requests\SearchNavigation\SearchNavigationRequest;
+
+const SORT_MAPPING = [
```

Comment:
I see this mapping across multiple files, please just add it to the constants file